pysm/data/presets.cfg

```diff
@@ -90,6 +90,20 @@ map_mbb_index = "pysm_2/dust_beta.fits"
 map_mbb_temperature = "pysm_2/dust_temp.fits"
 freq_ref_I = "545 GHz"
 freq_ref_P = "353 GHz"
+[d7]
+class = "HensleyDraine2017"
+map_I = "pysm_2/dust_t_new.fits"
+map_Q = "pysm_2/dust_q_new.fits"
+map_U = "pysm_2/dust_u_new.fits"
+unit_I = "uK_RJ"
+unit_Q = "uK_RJ"
+unit_U = "uK_RJ"
+freq_ref_I = "545 GHz"
+freq_ref_P = "353 GHz"
+rnd_uval = true
+seed = 4632
```

Comment:
@bthorne93 
Compare to PySM2: https://github.com/bthorne93/PySM_public/blob/master/pysm/nominal.py#L122